spec/features/attendances/delete_zoom_attendance_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'delete zoom attendance' do
+  include ApplicationHelper
+
+  before :each do
+    @user = mock_login
+    @module = create(:turing_module)
+    @test_attendance = create(:attendance_with_student_attendances, turing_module: @module)
```

Comment:
Change this to use the `:zoom_attendance` factory with the trait `:with_student_attendances`. I just checked the `attendance_with_student_attendances` factory and it doesn't look like it actually creates student attendances which is confusing. See [the FactoryBot docs](https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#using-traits) if you need help with how to use traits.

Reply:
thank you for the doc reference! 
updated it to: @test_attendance = create(:zoom_attendance, :with_student_attendances, turing_module: @module)
this attendance does have student attendances associated with it